src/styles/style.css

```diff
@@ -0,0 +1,204 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  font-family: Arial, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 716px;
+  padding-left: 10px;
+  gap: 1px;
+}
```

Comment:
The calendar days are not aligned properly. Consider using flexbox properties such as 'justify-content' and 'align-items' to align items in a container.